integration-test/2022-disputed-borders.py

```diff
@@ -126,3 +130,154 @@ def test_admin_level_3_other_place(self):
             z, x, y, 'boundaries', {
                 'id': 345678,
             })
+
+    def test_disputed_by_to_unrecognized_disputed_reference_line(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/13574166
+            dsl.way(13574166, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'disputed',
+                'disputed_by': 'RU;PK;IL;PS;SA;EG;ID;BD',
+                'name': '1949 Israeli-Syrian DMZ',
+                'ne:brk_a3': 'B16',
+                'ne_id': '1746705859;1746705871',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 13574166,
+                'disputed': True,
+                'kind:ru': 'unrecognized_disputed_reference_line',
+                'kind:pk': 'unrecognized_disputed_reference_line',
+                'kind:il': 'unrecognized_disputed_reference_line',
+                'kind:ps': 'unrecognized_disputed_reference_line',
+                'kind:sa': 'unrecognized_disputed_reference_line',
+                'kind:eg': 'unrecognized_disputed_reference_line',
+                'kind:id': 'unrecognized_disputed_reference_line',
+                'kind:bd': 'unrecognized_disputed_reference_line',
+                'kind': 'disputed_reference_line',
```

Comment:
In an alternate universe (see comment below), these might be `kind: disputed` and `kind:xx: unrecognized_disputed`.